tools/bumbershoot/idpassemble/idpassemble.xml

```diff
@@ -0,0 +1,122 @@
+        <param name="input" type="data" format="idpdb" label="Input File(s)" multiple="true"/>
+        <param name="max_fdr_score" type="float" label="Max FDR Score" value="0.05" />
+        <conditional name="filter_at_gene_level_condition">
+            <param name="filter_at_gene_level" type="boolean" truevalue="1" falsevalue="0" label="Filter at Gene Level" help="Apply filters at the gene level (i.e. 'min distinct peptides per gene group' instead of 'min distinct peptides per protein group')"/>
+            <when value="1">
+                <param name="min_distinct_peptides" type="integer" label="Min Distinct Peptides per Gene Group" value="2" />
+                <param name="min_spectra1" type="integer" label="Min Filtered Spectra per Gene Group" value="2" />
+                <param name="min_additional_peptides" type="integer" label="Min Additional Peptides per Gene Group" value="1" />
+            </when>
+            <when value="0">
+                <param name="min_distinct_peptides" type="integer" label="Min Distinct Peptides per Protein Group" value="2" />
+                <param name="min_spectra" type="integer" label="Min Filtered Spectra per Protein Group" value="2" />
+                <param name="min_additional_peptides" type="integer" label="Min Additional Peptides per Protein Group" value="1" />
+            </when>
+        </conditional>
+        <param name="min_spectra_per_distinct_match" type="integer" label="Min Filtered Spectra per Distinct Match" value="1" />
+        <param name="min_spectra_per_distinct_peptide" type="integer" label="Min Filtered Spectra per Distinct Peptide" value="1" />
+        <param name="max_protein_groups_per_peptide" type="integer" label="Max Protein Groups per Distinct Peptide" value="10" />
+        <!--<conditional name="advanced_options">
+            <param name="use_advanced_options" type="boolean" truevalue="true" falsevalue="false" label="Set Advanced Options" />
+            <when value="false" />
+            <when value="true">
+            </when>
+        </conditional>-->
+    </inputs>
+    <outputs>
+        <data format="idpdb" name="output" from_work_dir="output" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input" value="../../idpqonvert/test-data/201208-378803-mm.idpDB" />
+            <param name="max_fdr_score" value="0.05" />
+            <param name="filter_at_gene_level_condition.min_distinct_peptides" value="2" />>
+            <param name="filter_at_gene_level_condition.min_spectra" value="2" />
+            <param name="filter_at_gene_level_condition.min_additional_peptides" value="1" />
+            <param name="min_spectra_per_distinct_match" value="1" />
+            <param name="min_spectra_per_distinct_peptide" value="1" />
+            <param name="max_protein_groups_per_peptide" value="10" />
+            <output name="output" file="../../idpassemble/test-data/201208-378803-mm.idpDB" compare="sim_size" delta_size="500000" />
```

Comment:
VirtualBox doesn't like symlinks, and since I put my code inside the VM and share it with my Windows machine outside for editing, I couldn't go that route (I tried). I'll test it with --installed now. It works with planemo test.